Interview_Prep_App/native_java/retrofit/src/main/java/com/dreams/interviewprepapp/ui/main/SplashScreenActivity.java

```diff
@@ -1,28 +0,0 @@
-package com.dreams.interviewprepapp.ui.main;
```

Comment:
Ditto, please restore this file and simply change the manifest to use yours as launcher instead.

Reply:
Can you tell me what should i do. Should i create these files myself or should reset my commit before i deleted them.

Reply:
@prasad-more-12  Hi, I think you should reset your commit to before you deleted them, do a soft reset imo would be best.